lessons/networking/networking-networking-hardware.md

```diff
@@ -1 +1,97 @@
+
+1. [Home Router Components](#home-router-components)
+2. [Switch](#switch)
+3. [Router](#router)
+4. [Broadband Connection](#broadband-connection)
+5. [Wireless Access Point](#wireless-access-point)
+6. [Firewall](#firewall)
+
+# Lesson
+## Home Router Components
+Home routers typically contain following components:
+* Switch (Ethernet LAN)
+* Router (Ethernet WAN)
+* Broadband Connection
+* Wireless Access Point (Wi-Fi)
+* Firewall 
+
+![image](https://user-images.githubusercontent.com/18094862/179405675-5a1c6c1b-4f70-4b8e-89a4-982c522729c4.png)
+
+## Switch
+Switches facilitate sharing of resources on a Local Area Network (LAN). LAN includes all of the computers, printers, and servers in one location where all of the connected devices share information and usually these are all of the devices you own and trust. The highlighted section below represents a switch as a component of a home router. 
+
+![switch](https://user-images.githubusercontent.com/18094862/187009652-a687f468-f905-4f3f-b263-9cbec72a06b1.JPG)
+
+Ethernet cables are used to connect devices on the network. Below is the picture of an ethernet cable. 
+![image](https://user-images.githubusercontent.com/18094862/179407510-eecacf84-1513-4565-a9e4-55acc99d9fc0.png)
+
+## Router
+Routers connect to an Internet Service Provider (ISP) which is part of the Wide Area Network (WAN). WAN is network that can span the entire globe. An example of the largest WAN is the internet. The way we can get access to the internet is by connecting to the ISP. The ISP has connections to other routers on the internet creating a very large web-like network with many different paths between each source and destination. The ISP charges us to provide that service. 
+
+Just as a switch connects multiple devices to create a Local Area Network, a router connects multiple switches, and their respective networks, to form an even larger network. Home routers are different than enterprise routers used by ISPs within the internet primarily on their processing power, but conceptually they are the same.  
+
+Depending on the ISP, various different forms broadband connections are used which is described in next section. 
+![image](https://user-images.githubusercontent.com/18094862/179405602-b95cc6ef-db56-4b31-93b6-8354b759b69c.png)
+
+## Broadband Connection
+Broadband connection refers to the high-speed internet access technology which the ISP provides. The fastest broadband connection is fiber which is almost always preferable, but if that is not available then connections such as cable modem and DSL are preferred, wireless and satellite are less reliable, but in cases of rural networks they are the only ones available.  
```

Comment:
Please review for all acronyms. Unfortunately, because these are high school students and not industry people we cant assume anything. "what is DLS?"